### PR TITLE
LLM first transformers UT

### PR DESCRIPTION
**Pull request by Litchilitchy**
## Description
LLM first transformers UT

### CODE REVIEW

File: python/llm/test/inference/test_transformers_api.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from bigdl.llm.models import Llama, Bloom, Gptneox, Starcoder
+from bigdl.llm.utils import get_avx_flags
```
Comment (Oscilloscope98) [inline python/llm/test/inference/test_transformers_api.py:19]: No need for these two lines of imports

**Review by Oscilloscope98 — APPROVED** [review]
Others LGTM. Let's wait for the UT to pass

**Review by Oscilloscope98 — CHANGES_REQUESTED** [review]
Let's fix the error in UT

File: python/llm/test/inference/test_transformers_api.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        with torch.inference_mode():
+            st = time.time()
+            input_ids = tokenizer.encode(input_str, return_tensors="pt")
+            output = model.generate(input_ids, do_sample=False, max_new_tokens=32)
+            output_str = tokenizer.decode(output[0], skip_special_tokens=True)
+            end = time.time()
+        print('Prompt:', input_str)
+        print('Output:', output_str)
+        print(f'Inference time: {end-st} s')    
+
```
Comment (shane-huang) [inline python/llm/test/inference/test_transformers_api.py:52]: later you can submit more PRs to add Seq2Seq model and whisper.

**Review by shane-huang — APPROVED** [review]
This one LGTM.

**Comment by Oscilloscope98** [comment]
Merge for now. Will raise an PR to and some small fixes

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**